lib/base_bot.js

```diff
@@ -246,44 +246,49 @@ class BaseBot extends EventEmitter {
     const responseBody = {};
     return this.__validateSendOptions(sendOptions)
 
-    .then(() => {
-      let outgoingMiddlewarePromise;
-      if (this.master && !sendOptions.ignoreMiddleware) {
-        outgoingMiddlewarePromise = this.master.middleware.__runOutgoingMiddleware(
-          this, this.__associatedUpdate, outgoingMessage);
-      } else {
-        // don't actually go through middleware
-        outgoingMiddlewarePromise = Promise.resolve(outgoingMessage);
-      }
-      return outgoingMiddlewarePromise;
-    })
-    .then(() => {
-      responseBody.sentOutgoingMessage = outgoingMessage;
-      return this.__formatOutgoingMessage(outgoingMessage, sendOptions);
-    })
-    .then((rawMessage) => {
-      responseBody.sentRawMessage = rawMessage;
-      return this.__sendMessage(rawMessage, sendOptions);
-    })
-    .then((rawBody) => {
-      responseBody.raw = rawBody;
-      return this.__createStandardBodyResponseComponents(
-        responseBody.sentOutgoingMessage,
-        responseBody.sentRawMessage,
-        responseBody.raw);
-    })
-    .then((StandardBodyResponseComponents) => {
-      responseBody.recipient_id = StandardBodyResponseComponents.recipient_id;
-      responseBody.message_id = StandardBodyResponseComponents.message_id;
-      return responseBody;
-    })
-    .catch((err) => {
-      if (err === 'cancel') {
-        return 'cancelled';
-      }
+      .then(() => {
+        let outgoingMiddlewarePromise;
+        if (this.master && !sendOptions.ignoreMiddleware) {
+          outgoingMiddlewarePromise = this.master.middleware.__runOutgoingMiddleware(
+            this, this.__associatedUpdate, outgoingMessage);
+        } else {
+          // don't actually go through middleware
+          outgoingMiddlewarePromise = Promise.resolve(outgoingMessage);
+        }
+        return outgoingMiddlewarePromise;
+      })
+      .then(() => {
+        responseBody.sentOutgoingMessage = outgoingMessage;
+        return this.__formatOutgoingMessage(outgoingMessage, sendOptions);
+      })
+      .then((rawMessage) => {
+        responseBody.sentRawMessage = rawMessage;
+        // twitter_qr is a flag to identify if message is for twitter platform
+        let twitter_qr = false
+        if (outgoingMessage.quick_replies) {
+          twitter_qr = true
+        }
+        return this.__sendMessage(rawMessage, sendOptions, twitter_qr);
```

Comment:
The `rawMessage` object should contain enough information for you to know that this is a quick_reply in the `twitter-dm` package